### PR TITLE
Docs: restructure manage dashboards page

### PR DESCRIPTION
**Pull request by imatwawana**
This PR:

- Creates standalone **Troubleshoot dashboards** page
- Creates standalone **Import dashboards** page
- Removes import/export, discover, and troubleshooting content from **Manage dashboards** page
- General copy edits

TO DO

- [x] Clarify **Shared with me** details
- [x] Add canonicals

### CODE REVIEW

**Comment by imatwawana** [comment]
@jdbaldry - Can you help me with adding canonicals for the two new pages:
[Import dashboards](https://github.com/grafana/grafana/pull/81311/files#diff-d2e5fb637f5f4ab9ea0e2e05f84d83299eaa751ed78f6eb19a5e1a4323f67b80)
[Troubleshoot dashboards](https://github.com/grafana/grafana/pull/81311/files#diff-109be943b1b15feca5f24af2fd3901932e6730ab46f7158ea9e76bb2980b3afd)

File: docs/sources/dashboards/build-dashboards/import-dashboards/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+aliases:
+  - ../reference/export_import/
+  - export-import/
```
Comment (jdbaldry) [inline docs/sources/dashboards/build-dashboards/import-dashboards/index.md:4]:
```suggestion
aliases:
  - ../reference/export_import/
  - export-import/
canonical: https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/import-dashboards/
```

File: docs/sources/dashboards/manage-dashboards/index.md [hunk 2]
```diff
@@ -3,63 +3,52 @@ aliases:
   - ../features/dashboard/dashboards/
   - ../panels/working-with-panels/organize-dashboard/
   - ../reference/dashboard_folders/
-  - ../reference/export_import/
-  - ../reference/timerange/
-  - ../troubleshooting/troubleshoot-dashboards/
   - dashboard-folders/
   - dashboard-manage/
-  - export-import/
 keywords:
```
Comment (jdbaldry) [inline docs/sources/dashboards/manage-dashboards/index.md:8]:
```suggestion
canonical: https://grafana.com/docs/grafana/latest/dashboards/manage-dashboards/
keywords:
```

File: docs/sources/dashboards/troubleshoot-dashboards/index.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---
+aliases:
+  - ../troubleshooting/troubleshoot-dashboards/
+  - ../reference/timerange/
```
Comment (jdbaldry) [inline docs/sources/dashboards/troubleshoot-dashboards/index.md:4]:
```suggestion
aliases:
  - ../troubleshooting/troubleshoot-dashboards/
  - ../reference/timerange/
canonical: https://grafana.com/docs/grafana/latest/dashboards/troubleshoot-dashboards/
```

**Review by jdbaldry — COMMENTED** [review]
I've suggested canonicals for each of the files in question based upon the plan to use the OSS documentation as canonical URL.

I think the aliases need updating because the relative path they refer to is different from where they used to refer to in the old page.

I'll go through them and update my suggestions shortly.

**Review by axelavargas — APPROVED** [review]
Nice work, @imatwawana 🙌🏾 , the new structure looks great and makes more sense in my opinion.

**Review by ashharrison90 — APPROVED** [review]
lgtm 🙌

**Comment by imatwawana** [comment]
> I've suggested canonicals for each of the files in question based upon the plan to use the OSS documentation as canonical URL.
> 
> I think the aliases need updating because the relative path they refer to is different from where they used to refer to in the old page.
> 
> I'll go through them and update my suggestions shortly.

Thank you so much for your help! We still definitely need to do that in-depth lunch and learn on constructing aliases, lol.

**Pull request merged**